src/Synapse/Synapse/Common/HelpMessages.cs

```diff
@@ -440,5 +440,11 @@ public static class HelpMessages
         public const string AutoPauseTimer = "The period of inactivity in minutes before automatically pausing the sql pool.";
 
         public const string AutoResume = "Indicates whether the sql pool can automatically resume when connection attempts are made.";
+
+        public const string ManagedPrivateEndpointName = "The Synapse Managed Private Endpoint Name.";
+
+        public const string ManagedPrivateEndpointObject = "The Synapse Managed Private Endpoint Object.";
+
+        public const string DefaultVNetName = "Managed Virtual Network Name is default.";
```

Comment:
```suggestion
        public const string DefaultVNetName = "Managed Virtual Network Name is 'default'.";
```
suggest add quotes to avoid misguidence